connect-widget/app/src/pages/connect/index.tsx

```diff
@@ -16,10 +16,11 @@ const ConnectPage = () => {
   const [isLoading, setIsLoading] = useState(true);
   const [searchParams] = useSearchParams();
   const [isError, setIsError] = useState(null);
+  const [errorLink, setErrorLink] = useState(undefined);
+  const [errorTitle, setErrorTitle] = useState(undefined);
```

Comment:
NABD now, but any reason not to have a single object on state to represent these three props? Something like:

```
type DisplayError = {
  message: string;
  link: string;
  title: string;
}
...
const [error, setError] = useState<DisplayError | undefined>(undefined);
```

Reply:
Ah, that's how you do it! I tried to define them as an object together, but didn't quite get it in the end! haha thanks!